pallets/gear/src/runtime_api.rs

```diff
@@ -302,7 +302,7 @@ where
             None,
             None,
             payload,
-            BlockGasLimitOf::<T>::get() / 4,
+            BlockGasLimitOf::<T>::get() * 6,
```

Comment:
Isn't it a hack? Looks like magic.

Reply:
Not exactly. Just previous limitations became under required